src/lib/process/index.ts

```diff
@@ -1,16 +1,15 @@
 import { spawn, spawnSync, SpawnSyncOptions } from 'child_process'
 import { stripIndent } from 'common-tags'
 import * as path from 'path'
-import { format } from 'util'
 import { findFileRecurisvelyUpwardSync } from '../fs'
 import { log } from '../nexus-logger'
 
 /**
  * Log a meaningful semantic error message sans stack track and then crash
  * the program with exit code 1. Parameters are a passthrough to `console.error`.
  */
-export function fatal(template: string, ...vars: unknown[]): never {
-  log.fatal(format(template, ...vars))
+export function fatal(template: string, context?: Record<string, unknown>): never {
```

Comment:
Thoughts on:

```ts
export const fatal: typeof log.fatal = (...args) => {
  log.fatal(...args)
  process.exit(1)
}
```

Reply:
That would result in a type error as `log.fatal` has a return type of `void` vs `never`